Azure/azure-cli
# {App Service} Bugfix: Fix regression where az webapp create doesn't set specified runtime

**Description**  
Fixes #14409 
`az webapp create -g RG -n WebappName --plan Plan -r "runtime` doesn't set the runtime properly.
For example, 
`az webapp create -g RG -n WebappName --plan Plan -r "php|7.3"` sets the runtime to "php|5.x" instead of "php|7.3".

**Testing Guide**  
Run `az webapp create -g RG -n WebappName --plan Plan -r "php|7.3"`
Run `az webapp config show -g RG -n WebappName`

Should show phpVersion = 7.3, not 5.x

This checklist is used to make sure that common guidelines for a pull request are followed.
